biojava-structure/src/main/java/org/biojava/nbio/structure/Calc.java

```diff
@@ -538,12 +538,13 @@ public static final void transform(Atom atom, Matrix4d m) {
 	 * @param m
 	 */
 	public static final void transform(Group group, Matrix4d m) {
-		AtomIterator iter = new AtomIterator(group);
-
-		while (iter.hasNext()) {
-			Atom atom = iter.next();
+		for (Atom atom : group.getAtoms()) {
 			transform(atom, m);
-
+		}
+		for (Group altG : group.getAltLocs()) {
+			for (Atom atom : altG.getAtoms()) {
```

Comment:
Recursive call to `transform(g, m)` could also be possible here.

Reply:
Good point, now changed